packages/@uppy/core/src/Uppy.ts

```diff
@@ -483,17 +483,6 @@ export class Uppy<M extends Meta, B extends Body> {
     this.#emitter.emit(event, ...args)
   }
 
-  /** @deprecated */
```

Comment:
Waaaat? Why TypeScript? Let me have a look

Reply:
Running into the same problem now for `@uppy/transloadit`. `emit`'s work but `on` doesn't resolve.

Reply:
I don't think we need to make this more complex then we have to. Especially since in a couple days I will remove the deprecated events in `4.x` so what's the point in diving into it.

Reply:
What's extremely surprising is that it's working over there: https://github.com/transloadit/uppy/blob/d602d93829f1304326279da556577929564c3670/packages/@uppy/image-editor/src/ImageEditor.tsx#L22-L37